src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -173,3 +178,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.custom_command('create', 'pnp_create_key')
         g.custom_command('delete', 'pnp_delete_key')
         g.custom_command('update', 'pnp_update_key')
+
+    with self.command_group('iot central app', iot_central_sdk, client_factory=iot_central_service_factory) as g:
```

Comment:
```suggestion
    with self.command_group('iot central app', iot_central_sdk, client_factory=iot_central_service_factory, is_preview=True) as g:
```

Reply:
Because I have a little concern for command group naming rules and currently there is no command in `az iot central` group only one subgroup `app`, to catch the release we can add preview tag first and discuss later.